### DISCUSSION TITLE
Structurizr Lite - working with multiple workspaces

### DISCUSSION THREAD

**Discussion by JobaDiniz** [structurizr/structurizr.github.io#14] [Q&A] [answered]
Using the [docker version](https://docs.structurizr.com/lite/quickstart) of Lite, how can I create 2 workspaces and switch between them in the UI? If I create one file named `workspace.dsl` the Lite picks it and shows it. But how can I create more workspaces?

This doc mentions working with multiple workspaces but it's very vague: https://docs.structurizr.com/dsl/cookbook/workspace-extension/

**Comment by simonbrowndotje** [comment, marked as answer]
Structurizr Lite is only designed to work with a single workspace -> https://docs.structurizr.com/products#comparison
